Add DevExtreme Versions Topic (#8470)
DevExpress/devextreme-documentation: concepts/Common/DevExtreme Versions/10 Release Lifespans.md

> Each major DevExtreme release is supported for the following durations:

Reviewer: I made changes to the table, so need to update this introductory sentence accordingly. I also want to use some word that would indicate that we can deviate from this schedule. 

```suggestion
After we release a major DevExtreme release, we aim to publish minor updates according to the schedule below. 
```

<table class="dx-table">
    <tr>

>         <th>Update Type</th>

Reviewer: ```suggestion
        <th>Time since major version release</th>
```

> Each major DevExtreme release is supported for the following durations:
> 
> <table class="dx-table">
>     <tr>
>         <th>Update Type</th>
>         <th>Duration</th>
>     </tr>
>     <tr>
>         <td>Minor/Maintenance Updates</td>
>         <td>One Year</td>
>     </tr>
>     <tr>
>         <td>Security-Related Updates</td>
>         <td>Two Years</td>
>     </tr>
> </table>

Reviewer: The current version makes it seem like minor update and security updates are not the same thing (two different types of updates). It also looks like security issues are somehow only get shipped in the second year. 

I want to make it clear that those are all minor updates. And first we release them on a regular schedule. Then we release them as/if necessary. And first they contain both regular bug fixes and security fixes. But second year updates are only triggered because of security issues. 

```suggestion
<table class="dx-table">
   <tr>
	<th>Time since major release</th>
	<th>Minor release schedule</th>
	<th>Minor release content</th>
   </tr>
   <tr>
        <td>First 12 months</td>
        <td>Monthly</td>
        <td>Bug fixes and resolved security issues</td>
   </tr>
   <tr>
        <td>12-24 months</td>
        <td>As necessary</td>
        <td>Resolved security issues only</td>
   </tr>
</table>
```

Major release lifespans intersect. For instance, v25.1.8 was released at a later date than v25.2.3. However, v25.2.3 is the later version in terms of feature availability.

> For more information, refer to the following topic: [Supported Versions, Version History, and IDE Prerequisites](https://www.devexpress.com/support/versions.xml).

Reviewer: ```suggestion
For additional information, refer to the following web page: [Supported Versions, Version History, and IDE Prerequisites](https://www.devexpress.com/support/versions.xml).
```
